### DISCUSSION TITLE
Community infrastructure levy (CIL)

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#99] [Planning consideration]
# Community infrastructure levy (CIL)

The Community Infrastructure Levy (CIL) is a charge which can be levied by local authorities on new development. It only applies in areas where a local authority has consulted on and approved a charging schedule which sets out those charges

**Comment by colmjude** [comment]
## CIL Schedules

One part of the Community Infrastructure Levy planning consideration is the CIL schedule documents that authorities that have opted-in and adopted CIL publish.

These documents explain the rationale for adopting CIL and set out the rates that will be charged and how the rates will be calculated in future years.

We see collecting them as a stepping stone to having CIL data.

Therefore, we have designed a [cil-schedule schema](https://github.com/digital-land/specification/blob/main/content/dataset/community-infrastructure-levy-schedule.md?plain=1) that allows us to collect and publish these documents as data.

We then visited each planning authority website to see if we could find a CIL schedule. To date (2025-01-09) we have collected 139 [CIL schedule documents](https://www.planning.data.gov.uk/dataset/community-infrastructure-levy-schedule) and published them on planning.data.gov.uk

**Comment by colmjude** [comment]
# CIL Rates - what we know so far

CIL rates are set based on three core factors:

- Geographic area / zone
- Development type
- Conditions / modifiers

### Geographic area / zone

Some councils apply a single citywide rate (for example, Chelmsford).
Others divide their area into high-value vs. low-value zones (for example, Birmingham, Bracknell Forest).
Strategic development sites (e.g., large housing projects) often get reduced or zero CIL rates.

### Development type

Residential → Almost always charged, but rates vary.
Retail (Supermarkets, Large Stores) → Supermarkets and large retail often pay more than standard shops.
Retail (High Street, Shops, Restaurants, etc.) → lower or zero CIL
Employment (Offices, Industrial, Warehousing) → Usually £0/m² to encourage business growth.
Public Services (Education, Health, Community Buildings, etc.) → Almost always £0/m².

Examples: 

* In Chelmsford, convenience retail (food stores) is charged £150/m², while comparison retail (non-food) is £87/m²
* In Birmingham, only large supermarkets (>2,700m²) pay CIL, while smaller retail developments are £0/m²

### Conditions / modifiers

Conditions and modifiers include:

Size thresholds (e.g., retail over 2,700m² in Birmingham pays CIL, but smaller retail does not).
Number of dwellings (e.g., Bracknell Forest charges more for small sites but reduces rates for 15+ dwellings).
Type of occupancy (e.g., social housing, care homes, and self-build homes are often exempt).
Location-specific adjustments (e.g., Birmingham charges hotels in the city centre but not elsewhere).

Examples:

* Bracknell Forest → 1-14 dwellings in Northern Parishes pay £350/m², but 15+ dwellings pay £220/m² (page 3 of 7​bracknell_cil-charging-…).
* Birmingham → Student housing pays £69/m² unless it's in the Green Belt, where it is £0/m² (page 9 of 26​Birmingham_CIL_Charging…).
* Chelmsford → Self-build homes and social housing developments are exempt from CIL (page 2 of 8 ​chelmsford_cil-charging…).

**Comment by clairemiller2** [comment]
This summary looks good @colmjude. Conditions/modifiers are generally referred to as exemptions. Would also just clarify that the text under development type should be read as themes/initial findings, rather than rules.

Some strategic developments / large scale housing regeneration projects would have higher CIL rates. It depends of the local context.

Also, it is worth just flagging that CIL and S106 can be used together. S106 agreements must be mitigation schemes directly required as a result of growth (e.g. a development of 2,000 homes would require x forms of entry of additional primary school places. Whereas, CIL can be used to fund strategic priorities of the council, e.g. strategic transport schemes which aren't necessarily directly required as a result of growth at one or more development sites, or placemaking initiatives, such as active travel schemes or green infrastructure schemes. 

So once the LPA knows what infrastructure is required as a result of growth, as well as understanding the strategic aims of the council through planning policies, they will be able to understand the levels of contributions required, and the most appropriate mechanism for collecting contributions to fund these. E.g. if it is identified that the majority of infrastructure schemes required should be funded through S106, then the council might consider a lower CIL rate.

Depending on the priorities of the council, they would seek varying proportions of developer contributions. e.g. the more money collected through S106 for affordable housing will likely have an impact of the level of money able to be collected for infrastructure projects. Similarly, the level of CIL being sought is also likely to have implications of the level of S106 able to be collected. 

Technically, CIL is a non-negotiable levy, however, there are instances where developments may be given an exemption, or lower level to pay, to ensure that funding can be sought elsewhere. A good example of this would be where a developer is making significant improvements to an existing highway, which would lead to benefits beyond just that development. The charging authority might then reduce levels of CIL or S106 contributions being sought.

**Comment by drewhardy77** [comment]
One additional point to throw in on the indexation (how the charges will change in the future) is that it's crucial to extract the baseline index value at the point at which the CIL was adopted, as without this you can't calculate future/contemporary increases. For CIL schedules adopted in the past few years this is fairly straightforward but the further back you go the more opaque this information becomes.

**Reply by Exa-ChrisTrudgen** [reply]
Just to expand on this point slighty, as the oringialy used BCIS All In Tender Price Index was a commerical product from RICS, the CIL Authorites were not (and still cannot) publish their IC values due to the terms of the RICS licence if they adopted pre RICS CIL Index taking over in 2020. 

I remember hearing that RICS stopped people putting the IC values on their Liability/Demand notices, telling them to instead put the calculated index value (IC/IP) in place in the notice text to protect their product.

**Reply by drewhardy77** [reply]
Thanks Chris - very useful context.

**Comment by drewhardy77** [comment]
Additional point - development within London Boroughs is also subject to the Mayoral Community Infrastructure Levy on top of locally administered CIL (if any). This is split into borough-wide zones for differential charging

<img width="917" height="640" alt="image" src="https://github.com/user-attachments/assets/6541fcf9-c790-4790-ae39-102af97f9450" />


https://www.london.gov.uk/programmes-strategies/planning/implementing-london-plan/mayoral-community-infrastructure-levy
